format/src/main/scala/org/scalafmt/internal/TreeDocOps.scala

```diff
@@ -165,6 +165,10 @@ object TreeDocOps {
     SimpleExpr.wrap(lhs) + sep + rhs
   }
 
+  def dPath2(lhs: Tree, sep: Doc, rhs: Doc): Doc = {
+    print(lhs) + sep + rhs
```

Comment:
Don't we need to wrap Term.Apply qualifies in a syntactic group?

Reply:
It's already wrapped in the only place it was called. Now, I completely removed it and it's more obvious.